pkg/vsphere/extraconfig/encode.go

```diff
@@ -135,6 +136,15 @@ func encodeStruct(sink DataSink, src reflect.Value, prefix string, depth recursi
 	}
 }
 
+func isEncodableSliceElemType(t reflect.Type) bool {
+	switch t {
+	case reflect.TypeOf((net.IP)(nil)):
```

Comment:
Not sure how to do that since [reflect.Zero](https://golang.org/pkg/reflect/#Zero) takes in a Type, and I am trying to get a Type in the first place.